README.md

```diff
@@ -101,15 +101,30 @@ eight_points_guzzle:
             # plugin settings
             plugin:
                 oauth2:
-                    base_uri:       "http://om.dev"
-                    token_url:      "/app_dev.php/oauth"
+                    base_uri:       "https://example.com"
+                    token_url:      "/oauth/token"
                     client_id:      "test-client-id"
                     client_secret:  "test-client-secret" # optional
                     scope:          "administration"
                     grant_type:     "Sainsburys\\Guzzle\\Oauth2\\GrantType\\ClientCredentials"
                     auth_location:  "body"
 ```
 
+### Options
+
+| Key | Description | Example |
```

Comment:
What do you think about a separate row "Required"?

Reply:
yep, I thought about that, but there are dependencies. For example username and password are required if grant type is "PasswordCredentials" and so on. If I will add column required then there will be a long description and not simply "yes/no"... :disappointed:

Reply:
Tricky :/ Well, I don't have a perfect solution. One idea is to describe this by text, the other is to use different tables for each grant type.